FlashSpace/Features/Gestures/SwipeManager.swift

```diff
@@ -128,7 +130,7 @@ final class SwipeManager {
 
         if touchesCount == 0 {
             gestureFinished()
-        } else if touchesCount == 3 {
+        } else if touchesCount == swipeFingerCount, enableSwipeGesture {
```

Comment:
```suggestion
        } else if touchesCount == swipeFingerCount {
```

It's not needed. The whole swipe manager is turned on/off from `GesturesSettings`

Reply:
Then you can also remove `private var enableSwipeGesture: Bool { gesturesSettings.enableSwipeGesture }`